src/test/scala/uk.gov.nationalarchives.fileformat/LambdaTest.scala

```diff
@@ -15,21 +15,30 @@ class LambdaTest extends AnyFlatSpec with AWSSpec with FileSpec {
     outputQueueHelper.availableMessageCount should equal(1)
   }
 
-  "The update method" should "put one message in the output queue, delete the successful message and leave the key error message" in {
+  "The update method" should "reset the visibility of an invalid message so it can be retried" in {
```

Comment:
Should this be "... visibility of a _failed_ message ..."? Not sure the message is invalid, unless I'm missing something. Could be talking nonsense though.

Reply:
It fails the Circe parsing step because the consignment ID is an integer rather than a UUID.

I've reworded this test and the one just below to make it clearer why each message is a failure (i.e. parsing failure vs FFID failure).